commit a55658692b679521f8a656ab3b5e44d9fe5421dc (refs/changes/90/4490/3)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-07-20T14:18:57-07:00 (1 year, 3 months ago)
    
    TRITON-493 '--tree' option to sdc-imgadm, updates-imgadm, joyent-imgadm to show origin hierarchy (update sdc zone to latest sdc-imgadm)
    Reviewed by: Marsell Kukuljevic <marsell@joyent.com>
    Approved by: Marsell Kukuljevic <marsell@joyent.com>

diff --git a/package.json b/package.json
index dbf2f8b..ee9ecd9 100644
--- a/package.json
+++ b/package.json
@@ -10,7 +10,7 @@
         "smartdc": "git+https://github.com/joyent/node-smartdc.git#4308fd9",
         "manta": "git+https://github.com/joyent/node-manta.git#b277acf",
         "moray": "git+https://github.com/joyent/node-moray.git#b84ef0e",
-        "imgapi-cli": "git+https://github.com/joyent/sdc-imgapi-cli.git#v2.2.5",
+        "imgapi-cli": "git+https://github.com/joyent/sdc-imgapi-cli.git#v2.3.0",
         "bunyan": "1.3.3",
         "assert-plus": "0.1.5",
         "ldapjs": "git+https://github.com/mcavage/node-ldapjs.git#f213b3e",
